src/test/java/uk/gov/companieshouse/items/orders/api/service/DescriptionProviderServiceTest.java

```diff
@@ -17,11 +18,15 @@
 class DescriptionProviderServiceTest {
 
     private static final String COMPANY_NUMBER = "00006400";
+    private static final String ITEM_TYPE = "certificate";
```

Comment:
This has been opened up now to take other item types. Have we tested it with those other item types? Is that possible at this stage?

Reply:
We would have to test the description of each item. I've had a look and it appears we would have to add descriptions for scud and certificated copies in orders_description.yml? And we don't know this yet?

Reply:
Hence my earlier question about that descriptions file. Fine, if we don't know what those descriptions are then we can't do it! :)